src/components/Notification/Notification.styles.ts

```diff
@@ -0,0 +1,11 @@
+import { styled } from '@mui/material';
+
+export const Container = styled('div')(() => ({
+  maxWidth: 600,
+  rowGap: 10,
+  bottom: '20px',
```

Comment:
I think notifications will look wvisually better at the top of the screen, but it's just my personal opinion